src/main/java/digital/slovensko/autogram/Main.java

```diff
@@ -9,7 +10,11 @@ public static void main(String[] args) {
         AppStarter.start(args);
     }
 
-    public static String getVersion() {
-        return requireNonNullElse(System.getProperty("jpackage.app-version"), "dev");
+    public static Version getVersion() {
```

Comment:
mozno toto vsetko s Version mozem presunut do Version (`createCurrentVersion()`), a bude to davat viac zmysel ako taketo proxy

Reply:
Naopak, version nevie teraz o app nič a to je dobre.